.github/workflows/build.yml

```diff
@@ -0,0 +1,24 @@
+name: Build
+on:
+  pull_request:
+  push:
+    branches: [main] # Avoid duplicate builds on PRs.
+permissions:
+  contents: read
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [11, 8]
+        os: [windows-latest, ubuntu-latest, macos-latest]
```

Comment:
Nit: Not sure if we need to test builds on all OSes here, given the nature of this repo.

Reply:
write once, run everywhere?  i don't trust it 😉